src/v/kafka/server/handlers/topics/types.h

```diff
@@ -125,7 +128,6 @@ inline constexpr std::array<std::string_view, 20> allowlist_topic_noop_confs = {
   "follower.replication.throttled.replicas",
   "flush.messages",
   "file.delete.delay.ms",
-  "delete.retention.ms",
```

Comment:
vs log.cleaner.delete.retention.ms? not sure what the difference is.

Reply:
[In Kafka, `delete.retention.ms` is the topic property, while `log.cleaner.delete.retention.ms` is the server default property.](https://docs.confluent.io/platform/current/installation/configuration/topic-configs.html#delete-retention-ms)